assets/css/patientSignUp.css

```diff
@@ -213,5 +213,5 @@ textarea {
 }
 
 #matchButton{
-  rgba(19, 35, 47, 0.9)
+  rgba(19, 35, 47, 0.9);
 }
```

Comment:
Not sure what this symbol indicates. When hovering over it, it says "no newline at end of file". Maybe look into that to prevent a future error.